test/integration.test.ts

```diff
@@ -42,24 +34,24 @@ describe("integration tests", () => {
     });
   });
 
-  maybeConcurrentTest("should work without project", async () => {
-    const cwd = join("tmp", NO_PROJECT_TEST_NAME);
-    await mkdir(cwd, { recursive: true });
-
-    const result = await fluence({
-      args: ["run"],
-      flags: {
-        relay: multiaddrs[0]?.multiaddr,
-        env: fluenceEnv,
-        f: "identify()",
-        i: resolve(join("test", "_resources", "aqua", "smoke.aqua")),
-        quiet: true,
-      },
-      cwd,
-    });
-
-    const parsedResult = JSON.parse(result);
-    // Peer.identify() is supposed to return an object with air_version key
-    expect(parsedResult).toHaveProperty("air_version");
-  });
+  // maybeConcurrentTest("should work without project", async () => {
+  //   const cwd = join("tmp", NO_PROJECT_TEST_NAME);
+  //   await mkdir(cwd, { recursive: true });
+
+  //   const result = await fluence({
+  //     args: ["run"],
+  //     flags: {
+  //       relay: multiaddrs[0]?.multiaddr,
+  //       env: fluenceEnv,
+  //       f: "identify()",
+  //       i: "smoke.aqua",
+  //       quiet: true,
+  //     },
+  //     cwd,
+  //   });
+
+  //   const parsedResult = JSON.parse(result);
+  //   // Peer.identify() is supposed to return an object with air_version key
+  //   expect(parsedResult).toHaveProperty("air_version");
+  // });
```

Comment:
Why did you comment this? Just curious.
Is the test does nothing in the new code and you want to persist it for some reason?

Reply:
The reason is that I have a bit of a problem
I want cli to be able to recognize dependencies in a convenient way.
Normally it's fluence.yaml where you put all the dependencies
But if you are creating a library, you probably want just to directly use npm, so you can publish, pack and do other npm stuff in CI etc.
So added some code that would find a nearest package.json and use that dir for resolving dependencies in case fluence.yaml is not present

This test was for when there is no project, but I currently always have package.json top level so I can't have a no-project situation anymore with this design

Maybe you have any ideas on how to continue with this, or we can probably just not have this test, assume users always have fluence project when they compile aqua. I don't know

Reply:
restored this test by temporarelly removing package.json